pkg/version/version.go

```diff
@@ -16,15 +16,35 @@ package version
 
 import (
 	"fmt"
+	"os"
 	"runtime"
+	"syscall"
+
+	"github.com/cilium/ebpf"
+	"kmesh.net/kmesh/daemon/options"
+	"kmesh.net/kmesh/pkg/logger"
+)
+
+const (
+	NewStart = iota
+	Restart
+	Update
+	Reload
```

Comment:
what does reload mean

Reply:
mean we need to reload bpf map, it is restart